app/layout/scenarios/sidebar/wdpa/threshold/component.tsx

```diff
@@ -147,7 +147,26 @@ export const WDPAThreshold: React.FC<WDPAThresholdCategories> = ({
                     <Label ref={labelRef} theme="dark" className="mr-3 uppercase">Set the threshold for protected areas</Label>
                     <InfoButton>
                       <div>
-                        <h4 className="font-heading text-lg mb-2.5">Thresholding</h4>
+                        <h4 className="font-heading text-lg mb-2.5">What is a threshold?</h4>
+                        <div>
+
+                          Inside Marxan, planning units are considered as either
+                          protected
+                          or not protected. The threshold value represents a
+                          percentage of the area
+                          inside a planning unit. By setting the threshold you decide
+                          how much of a protected area needs to fall inside a
+                          planning unit to consider the whole planning unit
+                          as &quot;protected&quot;.
+                          <br />
```

Comment:
I don't like `<br />`. it would be better if we wrap the paragraphs into `<p>` tags and set in the `div` above the following class. `className="space-y-2"`. If you can do this for all the `<br />` you added I'll apreciate it. Remember to pull because I just merge it from develop and fix all the conflicts

Reply:
Oki doki!